ambari-server/src/main/java/org/apache/ambari/server/configuration/AmbariServerConfigurationKey.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ambari.server.configuration;
+
+import static org.apache.ambari.server.configuration.ConfigurationPropertyType.PASSWORD;
+import static org.apache.ambari.server.configuration.ConfigurationPropertyType.PLAINTEXT;
+
+/**
+ * Constants representing supported LDAP related property names
+ */
+public enum AmbariServerConfigurationKey {
```

Comment:
We may consider adding `LDAP_` just like we have (`SSO_`) prefix to these `LDAP` constants to help readers in client classes.

Reply:
I agree. I was trying to avoid too many unrelated changes in this patch since it was getting larger than I wanted it.